swap/src/monero.rs

```diff
@@ -240,7 +240,7 @@ pub trait CreateWalletForOutputThenLoadDefaultWallet {
 
 #[async_trait]
 pub trait OpenWallet {
```

Comment:
For Bitcoin, I got rid of most of these traits. Should we do the same thing for Monero?

Reply:
👍 Will do as follow up

Reply:
https://github.com/comit-network/xmr-btc-swap/pull/266